src/common/github-flavored-markdown.js

```diff
@@ -35,7 +35,7 @@ class GithubFlavoredMarkdown {
       }
     });
     extractor.use(replaceLink);
-    extractor.parse(markdown);
+    extractor.render(markdown);
```

Comment:
`parse` threw some errors when importing documents, probably because of invalid Markdown syntax